components/ui/Badge.js

```diff
@@ -22,18 +23,43 @@ const Badge = ({ technology }) => {
   const icon = getIcon(technology);
   if (!icon) return null; // Exit early - don't render anything
 
-  return (
+  const badgeContent = (
     <BadgeContainer
       onMouseEnter={() => setShowTooltip(true)}
       onMouseLeave={() => setShowTooltip(false)}
+      clickable={clickable}
     >
       <IconWrapper>{icon}</IconWrapper>
       {showTooltip && <Tooltip>{technology}</Tooltip>}
     </BadgeContainer>
   );
+
+  // If clickable, wrap in Link - if not, just return the badge
+  if (clickable) {
+    return (
+      <BadgeLink
+        href={`/technology/${encodeURIComponent(
+          technology.toLowerCase().replace(" ", "-")
+        )}`}
+      >
+        {badgeContent}
+      </BadgeLink>
+    );
+  }
+
+  return badgeContent;
 };
 
-const BadgeContainer = styled.div`
+const BadgeLink = styled(Link)`
+  text-decoration: none;
+  display: inline-block;
+`;
+
+//keeping "clickable" for styling logic and not passing it to DOM:
+
+const BadgeContainer = styled.div.withConfig({
+  shouldForwardProp: (prop) => prop !== "clickable",
+})`
```

Comment:
This looks like a Chat-GPT solution 😅. The cleaner and easier way is to prefix the property with a '$'. Like learned in the styled-components session (Reach out to the handout).